src/style.css

```diff
@@ -1,3 +1,93 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  color: black;
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  width: 1200px;
+  height: 768px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 0 40px;
+}
+
+.header img {
+  width: 40px;
+  height: 40px;
+}
+
+.img {
+  padding: 5px 0 0 10px;
+}
+
+.nav {
+  padding: 13px 0;
+}
+
+.items {
+  display: flex;
+  list-style-type: none;
+  text-transform: uppercase;
+}
+
+.item {
+  position: relative;
+  text-decoration: none;
+  color: black;
+  padding: 23px 5px;
+}
+
+.items li {
+  margin: 10px;
```

Comment:
Be consistent with your vertical margins (Add only top or only bottom margin, but not both). This helps avoid unexpected behavior due to margin collapse.